tutorials/sns-instances-notification-system/index.mdx

```diff
@@ -0,0 +1,292 @@
+   project_id = scaleway_account_project.sns_tutorial.id
+   name       = "publisher-server"
+   type       = "PLAY2-PICO"
+   image      = "debian_bookworm"
+   ip_id      = scaleway_instance_ip.publisher_public_ip.id
+   security_group_id= scaleway_instance_security_group.sns_www.id
+
+   # USER DATA TO RUN THE SERVER AT START-UP WITH A SCRIPT TO SET THE SNS ENVIRONMENT VARIABLES RELATED TO THE TOPIC
+   user_data = {
+     cloud-init = <<-EOF
+     #cloud-config
+     write_files:
+       - content: |
+           #!/bin/bash
+           export TOPIC_ARN="${scaleway_mnq_sns_topic.topic.arn}"
+           export AWS_ACCESS_KEY="${scaleway_mnq_sns_credentials.main.access_key}"
+           export AWS_SECRET_KEY="${scaleway_mnq_sns_credentials.main.secret_key}"
+         owner: root:root
+         path: /etc/profile.d/publisher-server_env.sh
+         permissions: '0755'
+     runcmd:
+       - apt-get update && apt-get install -y docker.io
+       - systemctl start docker
+       - systemctl enable docker
+       - chmod +x /etc/profile.d/publisher-server_env.sh
+       # Using 'source' command doesn't work in cloud-init runcmd because each command runs in a separate shell.
+       # this is why environment variables will be passed directly in the docker run command
+       - docker pull rg.fr-par.scw.cloud/sns-x-instance-tutorial/publisher-server:1.0
+       - |
+         . /etc/profile.d/publisher-server_env.sh
+         docker run -d --restart=always \
+           --name publisher-server \
+           -p 8081:8081 \
+           -e TOPIC_ARN=$TOPIC_ARN \
+           -e AWS_ACCESS_KEY=$AWS_ACCESS_KEY \
+           -e AWS_SECRET_KEY=$AWS_SECRET_KEY \
+           rg.fr-par.scw.cloud/sns-x-instance-tutorial/publisher-server:1.0
+   EOF
+   }
+ }
+```
+
+# Use the publisher and subscriber websites to test your setup
+
+1. Run the command `terraform apply`. You'll be prompt to enter your public SSH key. Paste the result of `cat ~/.ssh/id_ed25519.pub` and enter `yes`
+2. You need to confirm the subscription to the topic on the subscriber server. Go on the console, Instance, and copy the `subscriber_server` IP address.
+3. Then go on the home page of your subscriber server: ```http://<your_subscriber_ip_address>:8081```
+4. Click on `Confirm subscription`. If you have an error because the URL hasn't been received, you can reload the page. It should take less than 10 seconds to appear.
+5. Once you've arrived to the AWS subscription confirmation page, go back to the home page, and click on `Notifications`
+6. On the console, retrieve and copy the `publisher_server` IP address
+7. Go on the home page of your subscriber server: ```http://<your_publisher_ip_address>:8081```
+8. Click on one or another CPU behavior, and check on the notification page of your subscriber server if the notification message appeared.
+9. Once you're done testing, you can apply `terraform destroy` to clean and remove the project.
+
+[//]: # (# To go further)
```

Comment:
Are you planning to add some content here?

Reply:
For now I don't have much ideas. To apply this usecase in a real infrastructure, the best would be to use K8s. I plan to come back to it in the future, as soon as I have an idea.

Reply:
No problem - maybe in the meantime you could delete line 301